docs/list.md

```diff
@@ -93,6 +93,15 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **How to sign up**: Sign in with [Github here](https://devict-hacktoberfest.herokuapp.com/){:target="\_blank"} so they can track your PRs.
 - **Notes**: You need to create at least two PRs in any of the  Wichita developer community organizations listed [here](https://devict-hacktoberfest.herokuapp.com/){:target="\_blank"}
 
+### E
+
+#### **Encore**
+
+- **Swag**: A pack of 12 Encore stickers
+- **Requirements**: Build an app with Encore, tweet your Encore Flow diagram, or contribute to [encoredev/encore](https://github.com/encoredev/encore)
+- **How to sign up**: Pick up a [GitHub Issue](https://github.com/encoredev/encore/issues), or check out the [Quick Start] to build your first Encore app and once you're done post it in our [Show & Tell](https://encore.dev/docs/quick-start)!
```

Comment:
Hey @bih, I just noticed that it looks like your link to the Quick Start is empty and the Show & Tell link goes to the Quick Start page instead. Can you please provide a good link to the Show & Tell page? No need for another PR, just comment it here and I can take care of it. 👍 

Thanks!